var/spack/repos/builtin/packages/perl/package.py

```diff
@@ -45,11 +45,13 @@ class Perl(Package):  # Perl doesn't use Autotools, it should subclass Package
     # URL must remain http:// so Spack can bootstrap curl
     url = "http://www.cpan.org/src/5.0/perl-5.24.1.tar.gz"
 
+    version('5.26.2', 'dc0fea097f3992a8cd53f8ac0810d523')
```

Comment:
According to http://www.cpan.org/src/README.html, this is a maintenance release. Can you put it under the "Maintenance releases" section? 

I know we have a policy of always sorting versions, but Perl apparently doesn't have that policy (even numbered releases are maintenance, odd numbered releases are development). Maybe you can add a link to this page and a comment explaining this.